plugins/workspace-search/src/workspace_search.ts

```diff
@@ -502,7 +507,7 @@ export class WorkspaceSearch implements Blockly.IPositionable {
       // Search the whole string for collapsed blocks.
       blockText = block.toString();
     } else {
-      const topBlockText = [];
+      const topBlockText:any = [];
```

Comment:
I think this can be a string array.

Reply:
<img width="1078" alt="Screenshot 2023-11-01 at 14 11 29" src="https://github.com/google/blockly-samples/assets/91741581/b9cfaf1b-0c77-45fc-888e-33e3e17ec9df">
Actually, I got this warning. That's why I added `any` here.

Reply:
I think if you specificy it as `const topBLockText: string[] = [];` it will work! Can you give that a shot for me?

Reply:
Yes it worked.